packages/@lwc/shared/src/html-attributes.ts

```diff
@@ -49,24 +49,36 @@ export function isBooleanAttribute(attrName: string, tagName: string): boolean {
     );
 }
 
+// This list is based on https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes
 const GLOBAL_ATTRIBUTE = new Set([
-    'role',
     'accesskey',
+    'autocapitalize',
+    'autofocus',
     'class',
     'contenteditable',
     'contextmenu',
     'dir',
     'draggable',
-    'dropzone',
```

Comment:
`dropzone` is not mentioned [on MDN](https://developer.mozilla.org/en-US/search?q=dropzone). In [the HTML5 spec](https://html.spec.whatwg.org/multipage/obsolete.html), it is described as an "obsolete" attribute:

> **`dropzone` on all elements**
> Use script to handle the [dragenter](https://html.spec.whatwg.org/multipage/dnd.html#event-dnd-dragenter) and [dragover](https://html.spec.whatwg.org/multipage/dnd.html#event-dnd-dragover) events instead.

Removing it should not have an observable impact, except that users will get a warning whereas they didn't get a warning before.